core/storage/suggestion/gae_models_test.py

```diff
@@ -377,6 +377,76 @@ def test_get_translation_suggestion_ids_with_exp_ids_with_one_exp(self):
                 ['exp1'])), 1)
 
+    def test_get_exp_translation_suggestions_in_review_returns_limited_values(
+            self):
+        suggestion_models.GeneralSuggestionModel.create(
+            suggestion_models.SUGGESTION_TYPE_TRANSLATE_CONTENT,
+            suggestion_models.TARGET_TYPE_EXPLORATION,
+            'exp1', self.target_version_at_submission,
+            suggestion_models.STATUS_IN_REVIEW, 'author_3',
+            'reviewer_2', self.change_cmd, self.score_category,
+            'exploration.exp1.thread_6', self.translation_language_code)
+        suggestion_models.GeneralSuggestionModel.create(
+            suggestion_models.SUGGESTION_TYPE_TRANSLATE_CONTENT,
+            suggestion_models.TARGET_TYPE_EXPLORATION,
+            'exp1', self.target_version_at_submission,
+            suggestion_models.STATUS_IN_REVIEW, 'author_4',
+            'reviewer_2', self.change_cmd, self.score_category,
+            'exploration.exp1.thread_7', self.translation_language_code)
+
+        with self.swap(feconf, 'DEFAULT_QUERY_LIMIT', 1):
+            suggestions = (
+                suggestion_models.GeneralSuggestionModel
+                .get_translation_suggestions_in_review_with_exp_id('exp1'))
+
+        self.assertEqual(len(suggestions), 1)
+
+    def test_get_exp_translation_suggestions_in_review_for_resolved_suggestion_returns_no_items( # pylint: disable=line-too-long
+            self):
+        suggestion_models.GeneralSuggestionModel.create(
+            suggestion_models.SUGGESTION_TYPE_TRANSLATE_CONTENT,
+            suggestion_models.TARGET_TYPE_EXPLORATION,
+            'exp1', self.target_version_at_submission,
+            suggestion_models.STATUS_REJECTED, 'author_3',
+            'reviewer_2', self.change_cmd, self.score_category,
+            'exploration.exp1.thread_6', self.translation_language_code)
+        suggestion_models.GeneralSuggestionModel.create(
+            suggestion_models.SUGGESTION_TYPE_TRANSLATE_CONTENT,
+            suggestion_models.TARGET_TYPE_EXPLORATION,
+            'exp1', self.target_version_at_submission,
+            suggestion_models.STATUS_ACCEPTED, 'author_4',
+            'reviewer_2', self.change_cmd, self.score_category,
+            'exploration.exp1.thread_7', self.translation_language_code)
+
+        suggestions = (
+            suggestion_models.GeneralSuggestionModel
+            .get_translation_suggestions_in_review_with_exp_id('exp1'))
+
+        self.assertEqual(len(suggestions), 0)
+
+    def test_get_exp_translation_suggestions_in_review_for_non_translation_suggestion_returns_no_items( # pylint: disable=line-too-long
+            self):
+        suggestion_models.GeneralSuggestionModel.create(
+            suggestion_models.SUGGESTION_TYPE_ADD_QUESTION,
+            suggestion_models.TARGET_TYPE_SKILL,
+            'exp1', self.target_version_at_submission,
+            suggestion_models.STATUS_REJECTED, 'author_3',
```

Comment:
Suggest changing this & below to type IN_REVIEW to emphasize that there's only one condition difference between the tests rather than multiple (which helps focus the tests in this case).

Reply:
Ah, I was expecting to have STATUS_IN_REVIEW here (else the test doesn't make sense.). Thanks for raising this. 

Done!